scheduler/client/addrs.go

```diff
@@ -1,4 +1,4 @@
-package scootapi
+package client
```

Comment:
Putting addrs under client isn't clear to me. Now we have various packages across the repo importing scheduler/client - that's misleading

Reply:
fair - that does seem misleading to me too. will find a more suitable place for them.